apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -198,7 +198,7 @@ export default function Chat({
     <div className='flex flex-col h-full'>
       <div
         ref={containerRef}
-        className='flex flex-col gap-3 h-full overflow-y-auto pb-12'
+        className='flex flex-col gap-3 h-full overflow-y-auto custom-scrollbar pb-12'
```

Comment:
i think this scrollbar shows even if there is no scroll available, can you double check?

Reply:
## No run
<img width="686" alt="image" src="https://github.com/user-attachments/assets/74c02daf-2c4b-4186-82a5-04e43d08864b">

## Run short
<img width="617" alt="image" src="https://github.com/user-attachments/assets/91f34eba-53c3-4ac6-9f83-eaeffb9dd2a8">

## Run long enough for scroll 
<img width="600" alt="image" src="https://github.com/user-attachments/assets/ab7bbe48-b604-4248-8795-92a18909140f">